RawEventProcessor.cpp

```diff
@@ -687,13 +687,18 @@ bool RawEventProcessor::process_syscall_event(const Event& event) {
     std::shared_ptr<ProcessTreeItem> p;
     std::string cmdline;
 
-    if (!_cmdline.empty()) {
+    if (!_syscall.empty() && _syscall == "execve") {
```

Comment:
Please change `_syscall == "execve"` to `starts_with(_syscall, "execve")`. There are two execve syscalls "execve" and "execveat".

Reply:
Agreed.